src/vivarium/framework/values.py

```diff
@@ -110,6 +110,16 @@ def rescale_post_processor(value: NumberLike, time_step: pd.Timedelta):
         The annual rates rescaled to the size of the current time step size.
 
     """
+    if isinstance(time_step, Callable):
```

Comment:
I think I'm missing something, but how would `time_step` ever be anything but a `Callable`?

Reply:
I haven't added (or really fully thought through I think) the managing of swapping between local-functionality clocks and the old clocks without individual clocks, but the idea was that in the base case the `step_size` propery defined in `value` becomes the old `builder.time.step_size` which, when passed into the post-processor, is a `pd.Timedelta`.